package/usr/local/bin/hassbian-config

```diff
@@ -1,17 +1,18 @@
 #!/bin/bash
-
 SUITE_INSTALL_DIR=/opt/hassbian/suites
 SUITE_INSTALL_DB=/srv/homeassistant/hassbian/suite-states
```

Comment:
Since your rewriting the state code. Why not remove the old code and move the state file to where the old one was?

Reply:
Ups, removed all referances to old state functions now

Reply:
Moved control dir to srv/homeassistant/hassbian/control